cmatrix.c

```diff
@@ -630,6 +635,11 @@ if (console) {
                         while (i <= LINES && (matrix[i][j].val != ' ' &&
                                matrix[i][j].val != -1)) {
                             matrix[i][j].is_head = false;
+                            if(changes) {
+                                int dice = rand() % 8;
```

Comment:
Has a probability of ![](https://latex.codecogs.com/gif.latex?%5Cfrac%7B1%7D%7B8%7D) to change to another character.